src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -328,12 +328,44 @@ def link_to_project(prj, opts)
                      class: 'project', title: opts[:project])
   end
 
+  def link_to_remote_project(remote_prj, opts)
+    remote_sub_prj = resolve_remote_project_name(opts[:project])
+    base_url = URI.join(remote_prj.remoteurl, '/').to_s
+    out = opts[:short] ? ''.html_safe : 'project '.html_safe
+    out + link_to(opts[:project], "#{base_url}project/show/#{remote_sub_prj}",
```

Comment:
How do you know this works? This is the URL to the *API* which is not necessarily the same as the web ui url.

Reply:
yes, it works with api.opensuse.org, but with some interconnects it might not work. Only thing I have in mind would be adding a column to the database with the webui url and using this one if available. Would that make sense?

Reply:
It's already in `Configuration.obs_url` and there is already interconnect support for `Configuration`.